src/pages/checkout/payment-method.js

```diff
@@ -101,7 +101,8 @@ export class PaymentMethodPage extends Component {
   handlePayPalOrderResponse (order) {
     const payer = order.payer
```

Comment:
Could this line use destructuring?

```suggestion
    const { payer } = order
```